### ISSUE TITLE
Importing Box folders

### ISSUE THREAD

**Issue by fallenlucario** [kwsch/PKHeX#74]
Just a minor thing, not sure if it's intended but when I import a box folder (30 files) and I import it into let's say Box 9 it will say "Loaded 270 files to boxes." It will say the multiple of 30 for whatever box I import in the save. Even if I only import 30 files it will still say the box number x 30. 

Also, if i highlight for example two folders to try to import them simultaneously it will only import the first.  Just thought I'd report this. Thank you for the awesome application ^-^

**Comment by kwsch** [comment]
`if (Directory.Exists(files[0])) { loadBoxesFromDB(files[0]); return; }`

This line of code in f1-Main.cs only allows one folder to be imported at a time; it's to prevent people from dropping in tons of folders and overloading the program. Best to just do one at a time !

**Comment by fallenlucario** [comment]
Anytime! I use the application frequently so I'll report whatever minor oversights I find. For the most part it is a great program though so I can't thank you enough!

Ah I see, makes sense, 1 folder at a time is fine, I just wasn't sure if it was intentional is all.

One question! (This is just a minor thing I have) When I dump my boxes to separate boxes it will organize everything in a folder based on the dex number and all the other information. I was wondering if there could be an option to have it exported with the box number/slot in the beginning of the  file name, that way things wouldn't get out of the order they were set up in the program.

So for example if i had a Shiny Vivillon in box 19, slot 20 and  a non shiny in slot 21, If I chose to dump boxes the files in Box 19 could have  "[20]" and "[21]" in the beginning of their name in order to keep them in order in that new folder. 

Not sure how this could be implemented in a database dump with the DB folder having everything in one folder but i think the separate box dump idea would be easier. If you don't think the extra name addition is necessary that's alright but thank you for hearing me out! 

Lastly, when you save a pk6 file to the desktop, in the name of the file they always have a random string of characters, I thought it was the PID, what exactly are those characters for?

**Comment by kwsch** [comment]
Not slapping on what exact slot is exported from is advantageous for those who want to merge saves and ignore clones of what they already have. (Operating system will say you have duplicate files). If you want to export individual boxes, you can export a binary rip of the box with Save Box Data++ in the SAV tab. You can simply restore that box rip later if that's what you're intending to do.

The [characters at the end of the pkm is the Checksum&EncryptionConstant](https://github.com/kwsch/PKHeX/blob/a48642cb32a200836ff8d887c6757ff98d6dd121/Misc/PKX.cs#L388), which is enough data to sufficiently differentiate files.

**Comment by fallenlucario** [comment]
I see, thanks for the information!

Sorry to bother you with this however, I just found out about the QR feature which lets you import a pokemon to box 1 slot 1 of the game. I was wondering if there is any issues with n3ds?

I just tried it out multiple times on my n3ds, both in sysnand and emunand (both are 9.2.0-20u)
however the pokemon never shows up in the box. Does this have anything to do with the browsers being different in o3ds and n3ds?

the qr code worked fine on my o3ds. (9.2.0-20u)
